.github/workflows/android_deploy_production.yml

```diff
@@ -57,10 +57,22 @@ jobs:
       - name: Build Android apk
         run: flutter build apk --flavor production --release --build-number $GITHUB_RUN_NUMBER
 
+      - name: Get PR information
+        uses: 8BitJonny/gh-get-current-pr@2.2.0
+        id: PR
+
+      - name: Generate release notes
+        env:
+          PR_CONTENT: ${{ steps.PR.outputs.pr_body }}
+        id: generate-release-notes
+        run: |
+          echo "RELEASE_NOTE_CONTENT=$PR_CONTENT" >> $GITHUB_OUTPUT
+
       - name: Deploy Android Production to Firebase
         uses: wzieba/Firebase-Distribution-Github-Action@v1.5.0
         with:
           appId: ${{ vars.FIREBASE_ANDROID_APP_ID }}
           serviceCredentialsFileContent: ${{ secrets.FIREBASE_DISTRIBUTION_CREDENTIAL_JSON }}
           groups: ${{ vars.FIREBASE_DISTRIBUTION_TESTER_GROUPS }}
+          releaseNotes: ${{steps.generate-release-notes.outputs.RELEASE_NOTE_CONTENT}}
```

Comment:
@luongvo 
- `Should we use pr_title instead of `pr_body` to get the PR title?`: IMO, we should not. Since for the release production, we usually have a list of PRs to merge. Therefore, getting PR body is better

<img width="839" alt="Screenshot 2023-11-03 at 10 53 48 AM" src="https://github.com/nimblehq/flutter-templates/assets/88996251/81d57f1a-ff38-43d8-9bab-70e425ce0ea7">

-` Can we use it directly here?` : Yes, we can. Let me update all 🙏